win32ss/user/ntuser/menu.c

```diff
@@ -2992,10 +2992,8 @@ static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT fl
     /* We are off the right side of the screen */
     if (x + width > monitor->rcMonitor.right)
     {
-        if ((x - width) < monitor->rcMonitor.left || x >= monitor->rcMonitor.right)
-            x = monitor->rcMonitor.right - width;
-        else
-            x -= width;
+        /* Position menu at right edge of screen */
+        x = monitor->rcMonitor.right - width + 1;
     }
 
     /* We are off the left side of the screen */
```

Comment:
```suggestion
    /* We are off the left edge of screen */
```

Reply:
of *the* screen is more correct in terms of english language

Reply:
@HBelusca, I added "the" into both cases, but left the "side" and "edge" as they were. I believe these to be more correct based on English vernacular in current use. "Side" is a more general term such as "right side" and "left side" and "edge" refers more to a specific alignment such as at the "left edge" or "right edge" of the screen.